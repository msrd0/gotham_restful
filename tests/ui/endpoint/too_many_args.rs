#[macro_use]
extern crate gotham_restful;

#[derive(Resource)]
#[resource(read_all)]
struct FooResource;

#[read_all]
fn read_all(_id: u64) {}

fn main() {}
