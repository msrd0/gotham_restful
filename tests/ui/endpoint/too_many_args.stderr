error: Too many arguments
 --> tests/ui/endpoint/too_many_args.rs:9:4
  |
9 | fn read_all(_id: u64) {}
  |    ^^^^^^^^

error[E0412]: cannot find type `read_all___gotham_restful_endpoint` in this scope
 --> tests/ui/endpoint/too_many_args.rs:4:10
  |
4 | #[derive(Resource)]
  |          ^^^^^^^^ not found in this scope
  |
  = note: this error originates in the derive macro `Resource` (in Nightly builds, run with -Z macro-backtrace for more info)
